ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -13137,13 +13136,12 @@ private void updateDefaultTblProps(Map<String, String> source, Map<String, Strin
    *
    * @param tblProp
    *          property map
-   * @param isCTAS
    * @return Modified table property map
```

Comment:
running `create table b2 as select 1 aa` while the translator is disabled doesnt uncover exceptions ; however it results in  an EXTERNAL table which is also bad...